README.md

```diff
@@ -20,13 +20,52 @@ The service also delegates some responsibilities to other RESTful services e.g.
 
 #### Environment variables
 
-The following environment variables are required:
+The following environment variables are required - values given are for the dockerized local environment (cmc-integration-tests: https://github.com/hmcts/cmc-integration-tests):
+```
+GOV_NOTIFY_API_KEY=[YOU NEED THIS FROM SOMEONE WITH GOV NOTIFY ADMIN ACCESS]
```

Comment:
I would suggest not adding all these values here as then you're maintaining it in multiple places and this doesn't test that its accurate or maintained

Reply:
Agreed - where else we maintaining? I only added here because this is passed around a bit. I suppose I could just add something saying this list is incomplete, that will be enough :)

Reply:
cmc-integration-tests and in the helm chart